flutter/test/view_hierarchy/sentry_tree_walker_test.dart

```diff
@@ -0,0 +1,215 @@
+        (tester) async {
+      await tester.runAsync(() async {
+        await tester.pumpWidget(MyApp());
+
+        final first = _getFirstsentryViewHierarchy(instance);
+
+        expect(
+            true,
+            _findWidget(first, (element) {
+              return element.depth != null;
+            }));
+      });
+    });
+
+    testWidgets('returns a SentryViewHierarchyElement with a identifier',
+        (tester) async {
+      await tester.runAsync(() async {
+        await tester.pumpWidget(MyApp());
+
+        final first = _getFirstsentryViewHierarchy(instance);
+
+        expect(
+            true,
+            _findWidget(first, (element) {
+              return element.identifier == 'btn_1';
+            }));
+      });
+    });
+
+    testWidgets('returns a SentryViewHierarchyElement with X and Y',
+        (tester) async {
+      await tester.runAsync(() async {
+        await tester.pumpWidget(MyApp());
+
+        final first = _getFirstsentryViewHierarchy(instance);
+
+        expect(
+            true,
+            _findWidget(first, (element) {
+              return element.x != null && element.y != null;
+            }));
+      });
+    });
+
+    testWidgets('returns a SentryViewHierarchyElement with visibility',
+        (tester) async {
+      await tester.runAsync(() async {
+        await tester.pumpWidget(MyApp());
+
+        final first = _getFirstsentryViewHierarchy(instance);
+
+        expect(
+            true,
+            _findWidget(first, (element) {
+              return element.visible == true;
+            }));
+      });
+    });
+
+    testWidgets(
+        'does not return a SentryViewHierarchyElement without visibility',
+        (tester) async {
+      await tester.runAsync(() async {
+        await tester.pumpWidget(MyApp());
+
+        final first = _getFirstsentryViewHierarchy(instance);
+
+        expect(
+            false,
+            _findWidget(first, (element) {
+              return element.visible == false;
+            }));
+      });
+    });
+
+    testWidgets('returns a SentryViewHierarchyElement with alpha',
+        (tester) async {
+      await tester.runAsync(() async {
+        await tester.pumpWidget(MyApp());
+
+        final first = _getFirstsentryViewHierarchy(instance);
+
+        expect(
+            true,
+            _findWidget(first, (element) {
+              return element.alpha == 0.5;
+            }));
+      });
+    });
+
+    testWidgets('does not return a SentryViewHierarchyElement with private key',
+        (tester) async {
+      await tester.runAsync(() async {
+        await tester.pumpWidget(MyApp());
+
+        final first = _getFirstsentryViewHierarchy(instance);
+
+        expect(
+            false,
+            _findWidget(first, (element) {
+              return element.identifier == '_btn_3';
+            }));
+      });
+    });
+  });
+}
+
+SentryViewHierarchyElement _getFirstsentryViewHierarchy(WidgetsBinding instance) {
```

Comment:
L: _getFirstsentryViewHierarchy -> _getFirstSentryViewHierarchy
I'm not sure if the 's' is lower case on purpose or not?

Reply:
typo, fixed, thanks.